docs/api.md

```diff
@@ -689,8 +689,10 @@ Parameter            | Type         | Optional | Default | Description
 iterations           | int          | yes      | N/A     | Max number of solver's iterations
 snapshot             | int          | yes      | N/A     | Iterations between model snapshots
 snapshot_prefix      | string       | yes      | empty   | Prefix to snapshot file, supports repository
-solver_type          | string       | yes      | SGD     | from "SGD", "ADAGRAD", "NESTEROV", "RMSPROP", "ADADELTA", "ADAM",  "AMSGRAD", "ADAMW", "SGDW", "AMSGRADW" (*W version for decoupled weight decay)
+solver_type          | string       | yes      | SGD     | from "SGD", "ADAGRAD", "NESTEROV", "RMSPROP", "ADADELTA", "ADAM",  "AMSGRAD", "RANGER", "ADAMW", "SGDW", "AMSGRADW" (*W version for decoupled weight decay)
```

Comment:
add RANGER_PLUS for the one with gradient centralization and adabelief ?

Reply:
ok